src/main/java/com/meraphorce/mapper/IMapper.java

```diff
@@ -0,0 +1,8 @@
+package com.meraphorce.mapper;
+
+public interface IMapper<T, D> {
+	public T toEntity(D dtoValue);
+	
+	public D toDTO(T entityValue);
+		
+}
```

Comment:
Add new line.
Ending a file with a newline (also known as a line break or an end-of-line (EOL) character) is a convention rooted in the definitions of Unix-based systems and is considered a best practice for several practical reasons in software development